### ISSUE TITLE
gh namespace?

### ISSUE THREAD

**Issue by nilsdeppe** [sxs-collaboration/spectre#425]
Should we have a `gh` or `gharmonic` namespace? Or maybe a `gr` or `relativity` namespace? (There's not much to compute in SR, so there should be no confusion). We could then park all the functions in the `PointwiseFunctions/GeneralRelativity` functions there.

Thoughts from others?

**Comment by nilsdeppe** [comment]
After seeing the tests and likely names for variables this should be done. Basically, the `compute_` prefix should be replaced with a namespace. This should be done before #423 is moved much closer to merging.

**Comment by markscheel** [comment]
Should this be done in a separate PR from #423?

**Comment by markscheel** [comment]
Should the namespace be `gr` or `gh`?  Many more functions would be encompassed by `gr`.

**Comment by nilsdeppe** [comment]
@osheamonn and I discussed this earlier today and we were thinking `gr` since these functions are, as you say, not tied to GH. It would be nice (as in no conflicts) if you prepended a commit to your PR that makes this change.

**Comment by nilsdeppe** [comment]
Addressed with #423
